Doppler.Sap/Models/DopplerUserDTO.cs

```diff
@@ -32,6 +32,7 @@ public class DopplerUserDto
         public string BillingCountryCode { get; set; }
         public int? BillingStateId { get; set; }
         public int? PlanType { get; set; }
+        public int PaymentMethod { get; set; }
```

Comment:
I have a doubt with Relay, I think this parameter is no passed in Relay, so If this property is null, the system returns a HTTP 400 because is an int, maybe can be an` int?`

Reply:
Definitly, I checked in `Relay` and this property is not passed to` Doppler.Sap`. Because of that, I tested this scenario and it worked fine. `DopplerUserDto` have NULL in` PaymentMethod` and in mapping method is sets DEFAULT value for `PayTermsGrpCode`.

Reply:
Sorry, I checked the scenario without PaymentMethod , but not with PaymentMethod = null. I think in Relay PaymentMethod is not included in the request content. Do you think that its possible? I can fix that anyway, just in case.

Reply:
if you do not pass PaymentMethod in the payload, inside controller PaymentMethod = 0, I checked in Relay the payment method, so I think that is not necessary to fix this problem